docs/modules/ROOT/pages/configure-diagnostic-logging.adoc

```diff
@@ -0,0 +1,62 @@
+* fileNamePrefix: add a custom prefix.
+* pluginProperties: pass the same diagnostics system properties shown above (e.g. enable plugins or adjust their periods).
+* autoOffTimerInMinutes: configure automatic turn-off (see below).
+* pvcName: bind diagnostics to a PVC (see below).
+
+== Forward diagnostics to a logging framework
+
+Instead of file output, you can forward diagnostics to your logging framework by setting `spec.diagnostics.outputType=LOGGER`
+
+Then configure the com.hazelcast.diagnostics logger (e.g. a Log4j2 logger and rolling appender) to control size and rollover using time/size policies. For information, see https://docs.hazelcast.com/hazelcast/latest/maintain-cluster/monitoring#diagnostics-using-logging-frameworks[Diagnostics using Logging Frameworks].
+
+== Configure auto turn-off
+
+To prevent diagnostic logs from running indefinitely and having a performance impact on the cluster, set the Operator's `autoOffTimerInMinutes` to automatically disable diagnostic logging after the specified number of minutes. See https://docs.hazelcast.com/operator/latest-snapshot/api-ref#diagnostics[Diagnostics].
+
+You can combine this with rolling limits:
+
+* Operator diagnostics: maxRolledFileSize, maxRolledFileCount.
+* Hazelcast properties: hazelcast.diagnostics.max.rolled.file.size.mb and hazelcast.diagnostics.max.rolled.file.count.
+
+NOTE: When diagnostics logging is enabled with auto turn-off, members that join the cluster later will start their own timers from zero. As a result, earlier members may stop diagnostics before the last joiner, leading to a temporary inconsistency. To keep the reported state accurate, Operator restarts its diagnostics auto turn-off timer from zero whenever a new member joins while diagnostics is enabled. This ensures Operator shows diagnostics as enabled as long as at least one member is still logging. 
+
+== Persist diagnostics to a PVC
+
+To retain diagnostics across pod restarts and isolate them from container stdout, mount a Persistent Volume Claim and direct diagnostics there:
+
+* Set the diagnostics output to files and point the directory to a mounted path using hazelcast.diagnostics.directory.
+* In the Operator, specify `pvcName` in the diagnostics config to mount a PVC for diagnostics. This instructs the Operator to use the named PVC for diagnostics storage (the detailed mounting YAML is not included in the sources).
+
+NOTE: System log diagnostics (cluster activity and optional partition migration details) can be enabled/tuned via `hazelcast.diagnostics.systemlog.enabled` and `hazelcast.diagnostics.systemlog.partitions` if needed. See https://docs.hazelcast.com/hazelcast/latest/maintain-cluster/monitoring#systemlog[SystemLog].
+
+== Configure diagnostics via system properties
+
+// question - If you set diagnostics using system properties, does this interfere with the settings configured via the CR? Is there anything else we need to tell them here?
```

Comment:
If the diagnostic is enabled via system properties, it won't be possible to turn it off using the CR config as system properties have higher priority